src/utils.zig

```diff
@@ -1530,6 +1530,228 @@ pub const BinaryWriter = struct {
 	}
 };
 
+const ReadWriteTest = struct {
+	fn init() void {
+		main.initThreadLocals();
+	}
+	fn deinit() void {
+		main.deinitThreadLocals();
+	}
```

Comment:
Instead of requiring these for every test (and every future test for that matter), how about just assigning the `stack`/`globalAllocator`s in main.zig the testing allocator if we are in a test build. You can use `if(builtin.is_test)` for this.

Reply:
So I ended up doing:

```zig
const ReadWriteTest = struct {
	pub var testingAllocator = main.heap.ErrorHandlingAllocator.init(std.testing.allocator);
	pub var allocator = testingAllocator.allocator();
```

To completely avoid touching `main.zig`, is that alright?

Reply:
For now it's fine, but this issue will come back if we actually test things that use these allocators. I made an issue for it: #1286